post-scan-actions/aws-python-teams-notification/README.md

```diff
@@ -0,0 +1,191 @@
+# Post Scan Action - MS Teams Notification
+
+After a scan occurs and a malicious file is detected, this example Lambda function sends out a notification to Microsoft Teams using Amazon Simple Notification Service.
+
+## Prerequisites
+
+1. **Install supporting tools**
+    Do one of the following:
+        - Install the AWS command line interface (CLI). See [Installing the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html) for details.
+        - Install GNU Make. See [GNU Make](https://www.gnu.org/software/make/) for download information.
+2. **Configure Microsoft Teams Webhook Connector**
+    - Create a channel for a Team
+    - Next to the team name click the '...'
```

Comment:
- Click the **...** ellipsis next to the team name.

Reply:
Done.